.storybook/main.js

```diff
@@ -71,11 +67,41 @@ module.exports = {
     },
     resolve: {
       ...storybookConfig.resolve,
-      alias: appConfig.resolve.alias,
+      alias: {
+        ...appConfig.resolve.alias,
+        ...(isEmbeddingSdkPackageInstalled && {
+          "embedding-sdk": require.resolve("@metabase/embedding-sdk-react"),
+        }),
+      },
       extensions: appConfig.resolve.extensions,
     },
   }),
 };
 
 const isCSSRule = rule => rule.test.toString() === "/\\.css$/";
 const isSvgRule = rule => rule.test && rule.test?.test(".svg");
+
+function resolveEmbeddingSdkPackage() {
+  let isEmbeddingSdkPackageInstalled = false;
+  let embeddingSdkVersion;
+
+  try {
+    embeddingSdkVersion =
+      require("@metabase/embedding-sdk-react/package.json").version;
+    isEmbeddingSdkPackageInstalled = true;
+  } catch (err) {
+    const sdkPackageTemplateJson = fs.readFileSync(
+      path.resolve(
+        "./enterprise/frontend/src/embedding-sdk/package.template.json",
+      ),
+      "utf-8",
+    );
+    const sdkPackageTemplateJsonContent = JSON.parse(sdkPackageTemplateJson);
+    embeddingSdkVersion = JSON.stringify(sdkPackageTemplateJsonContent.version);
```

Comment:
Ah I see, thanks for the explanation!

Reply:
I'm not sure I understand any of these.
1. This is the storybook's webpack config, so it shouldn't have anything to do with our code. I tested it just in case and we're still fine![image](https://github.com/user-attachments/assets/f7dea4cd-e1e2-4688-9dce-9fe82e699dd8)
2. I could only find `EnvironmentPlugin` in this file, but not `DefinePlugin`

Anyway, I don't think we have to worry about the actual version sent to Metabase getting quoted unexpectedly.